### DISCUSSION TITLE
Should we prevent the Probot Stale app to close PRs/issues?

### DISCUSSION THREAD

**Discussion by Panquesito7** [TheAlgorithms/C-Plus-Plus#1467] [Ideas] [answered]
Currently, there are 400+ PRs, most of them being stale/abandoned. The [Probot Stale](https://github.com/marketplace/stale) app is closing many issues and PRs. I think we should just mark the PR/issue as abandoned/stale and keep it as-is. If the author wants to come back and re-work the PR/issue, it'd be better if the PR/issue isn't closed.

What do you think about this? 🙂

**Comment by Panquesito7** [comment]
@ayaankhan98 @cclauss.

**Reply by ayaankhan98** [reply]
It does not directly closes the PR, first it mark them stale for about two weeks or so. And if the author is not responding to that for 2  weeks. i think it's fine to close them. After all if he/she is intrested in continuing that work he/she can respond with a single piece of message when he/she will continue. But if things remains as it as for about two weeks it better to close them. And even after that if author wishes to reopen there is no problem with that we will consider that.

**Reply by Panquesito7** [reply]
Thanks for your comments; I think the same too. _Offtopic: Preferably, when making opinions, click on the "Answer" button below and write your reply there. A reply is used when, for example, you write your opinion and I am replying (or writing my thoughts) about your opinion. If you do this, I'll be able to mark your opinion as an answer, but I cannot mark replies as an answer. Hope I get to understand._ 🙂

**Reply by ayaankhan98** [reply]
Yup got that. Earlier i did not bothered about all this, I just presented my opinion. Well i will remember now onwards :)

**Comment by cclauss** [comment, marked as answer]
Other language repos use this tool so I see no reason not to use it if maintainers can not keep up with the contributions.
